ui/app.yaml

```diff
@@ -55,7 +55,7 @@ handlers:
         https://api-dot-all-of-us-workbench-test.appspot.com
         https://api-dot-all-of-us-rw-staging.appspot.com
         https://api-dot-all-of-us-rw-perf.appspot.com
-        https://api-dot-all-of-us-rw-stable.appspot.com
```

Comment:
This is stable, not staging.  Can you confirm that this is what's happening:
1. the old stable API was not removed earlier, for release-safety reasons
2. stable can now be removed
3. now the old staging API remains, also for release-safety reasons

Reply:
Yes, that is what I was thinking. Keep old staging host for now. But we are okay to delete old stable hostname